onboarding_project/.gitignore

```diff
@@ -0,0 +1,5 @@
+#ignore the binary files
+*.pyc
+
+#ignore xmls
+*.xml
```

Comment:
This red thing means that there's no newline at the end of the file. Add a new line.